src/v2/Components/LotTimer.tsx

```diff
@@ -30,6 +33,7 @@ export const LotTimer: React.FC<LotTimerProps> = ({ saleArtwork }) => {
       <Text variant="md" color={"black60"}>
         {saleArtwork.formattedStartDateTime}
       </Text>
+      {extendedBiddingInfoLabel}
```

Comment:
👍🏻 good suggestion

Reply:
my thought process on this was that this way I could write a test for the function rather than have to go through the component rendering etc which I assumed would be slower, but I could be wrong about that.

Reply:
`boolShouldRender && <ConditionallyRenderedComponent />` is a pretty standard first-class way of that kind of thing, so I would stick to that in this top-level component